core/src/test/java/org/apache/calcite/test/BookstoreSchema.java

```diff
@@ -90,16 +96,50 @@ public Author(int aid, String name, Place birthPlace, List<Book> books) {
   /**
    */
   public static class Place {
-    public final Coordinate coords;
     public final String city;
     public final String country;
+    public final double elevationInMeters;
+    public final Coordinate coords;
+    public final Area area;
```

Comment:
I tested against elevationInMeters and Area.number.

The reason why they are necessary because the targeted type has to be different than the first field's type of the current level. Here, elevationInMeters type has to be different than city, and area.number has to be different than areaUnit.